caluma/caluma_workflow/models.py

```diff
@@ -167,13 +168,14 @@ class WorkItem(UUIDModel):
     STATUS_COMPLETED = "completed"
     STATUS_CANCELED = "canceled"
     STATUS_SKIPPED = "skipped"
+    STATUS_SUSPENDED = "suspended"
 
-    STATUS_CHOICES = (STATUS_READY, STATUS_COMPLETED, STATUS_CANCELED)
     STATUS_CHOICE_TUPLE = (
         (STATUS_READY, "Task is ready to be processed."),
         (STATUS_COMPLETED, "Task is done."),
         (STATUS_CANCELED, "Task is cancelled."),
         (STATUS_SKIPPED, "Task is skipped."),
+        (STATUS_SUSPENDED, "Task is suspended."),
```

Comment:
Just noticed due to this change's effect in the migrations: This is on the `WorkItem` model, not `Task`, thus should have work item in the description. Maybe add a drive-by fix commit?

Reply:
Absolutely, that bothered me as well. Fixed in bfed4981b73acfbe49bb1ab4644b5742d63058b3